src/wms/src/test/java/org/geoserver/wms/featureinfo/RenderingBasedFeatureInfoTest.java

```diff
@@ -353,11 +353,30 @@ public void testMapWrapping() throws Exception {
             // with wrapping enabled we should get the giant polygon on the other side too
```

Comment:
Added. Based on git log, I think it dates back to 2013 - the original commit looks like all new tests in that file, rather than relocating some existing code.

However since *you* added it, feel free to provide more context.

(I didn't rebase, just to make the review of the changes easier - feel free to squash).